machine_test.go

```diff
@@ -91,6 +91,10 @@ func (s *fakeSupervisor) Hang(ctx context.Context, _ struct{}, _ *struct{}) erro
 	return ctx.Err()
 }
 
+func (s *fakeSupervisor) Register(ctx context.Context, _ service, _ *struct{}) error {
+	// Tests expect that all registrations succeeed.
+	return nil
+}
```

Comment:
```suggestion
}

```

Reply:
Oops; I missed this. It's fixed in https://github.com/grailbio/bigmachine/pull/64 .